src/crypto.js

```diff
@@ -96,7 +96,6 @@ class Crypto {
     console.table(data);
   }
 
-
   async monoExchange() {
```

Comment:
This class was made to do API requests to cryptocompare.com, you need to create new class or add these features as functions and export them.